app/graphql/types/correction_type.rb

```diff
@@ -0,0 +1,20 @@
+module Types
+  class CorrectionType < Types::BaseObject
+    class CorrectionKind < Types::BaseEnum
```

Comment:
```suggestion
    class CorrectionTypeType< Types::BaseEnum
```

C'est un peu moche, mais on utilise la même technique pour d'autres `enum` (par exemple, AddressType). Je préfère ne pas introduire un nouveau terme (« kind ») dans les interfaces d'API.

Reply:
résolu par le renommage `kind => reason` : on a maintenant `CorrectionReasonType`